Tasks/AzureResourceGroupDeployment/operations/WinRMExtensionHelper.ts

```diff
@@ -395,13 +395,13 @@ export class WinRMExtensionHelper {
                 tl.debug("Addition of extension completed for vm: " + vmName);
                 if (result["properties"]["provisioningState"] != 'Succeeded') {
                     tl.debug("Provisioning State of CustomScriptExtension is not suceeded on vm " + vmName);
-                    reject(tl.loc("ARG_SetExtensionFailedForVm", this.resourceGroupName, vmName, result));
+                    reject(tl.loc("ARG_SetExtensionFailedForVm", this.resourceGroupName, vmName, JSON.stringify(result)));
                     return;
                 }
                 var validStatus = await this.ValidateExtensionExecutionStatus(vmName, dnsName, extensionName, location, _fileUris);
                 if (!validStatus) {
                     tl.debug("WinRMCustomScriptExtension is not valid on vm " + vmName);
-                    reject(tl.loc("ARG_SetExtensionFailedForVm", this.resourceGroupName, vmName, result));
+                    reject(tl.loc("ARG_SetExtensionFailedForVm", this.resourceGroupName, vmName, JSON.stringify(result)));
```

Comment:
Will it make sense here. I think we should attempt to print the stderr result message